docs/reference/rest-api/common-parms.asciidoc

```diff
@@ -70,6 +70,56 @@ tag::cat-v[]
 to `false`.
 end::cat-v[]
 
+tag::doc-pipeline[]
+`pipeline`::
+(Optional, string) The ID of the pipeline to use to preprocess incoming
+documents.
+end::doc-pipeline[]
+
+tag::doc-refresh[]
+`refresh`::
+(Optional, enum) If `true` then refresh the affected shards to make this
```

Comment:
Current wording could imply the user is expected to refresh the shards. We may want to explicitly state ES will refresh if `true`.

Reply:
Fixed